client/components/domains/register-domain-step/index.jsx

```diff
@@ -387,13 +387,17 @@ const RegisterDomainStep = React.createClass( {
 
 				if ( abtest( 'domainSuggestionNudgeLabels' ) === 'withLabels' ) {
 					const isFree = ( suggestion ) => ( suggestion.is_free === true ),
+						strippedDomain = domain.replace( / \./g, '' ),
```

Comment:
Stripped the dot too, think it's a good idea if someone uses them and inputs an invalid domain. But don't worry too much about it. We're testing reinforcement.